sections/06_discussion.md

```diff
@@ -49,6 +49,86 @@ with only a couple GPUs.*
 *Some of this is also outlined in the Categorize section.  We can decide where
 it best fits.*
 
+Efficiently scaling deep learning is challenging, and there is a high
+computational cost (e.g., time, memory, energy) associated with training neural
+networks and using them for classification. For these reasons, neural networks
+have only recently found widespread use [@tag:Schmidhuber2014_dnn_overview].
+For biologists, such problems are further complicated by the immense size of
+most biological datasets.
```

Comment:
The disk space problem is an issue, but not the only one. I agree that its current placement might alienate discussion of other motivating issues (e.g., "wide" datasets). Perhaps I should leave this sentence out entirely?